client/ayon_core/hosts/houdini/plugins/publish/collect_files_for_cleaning_up.py

```diff
@@ -0,0 +1,85 @@
+import pyblish.api
+import os
+from ayon_core.pipeline import AYONPyblishPluginMixin
+from ayon_core.hosts.houdini.api import lib
+
+
+class CollectFilesForCleaningUp(pyblish.api.InstancePlugin,
+                                AYONPyblishPluginMixin):
+    """Collect Files For Cleaning Up.
+    
+    This collector collects output files
+    and adds them to file remove list.
+
+    CAUTION:
+        This collector deletes the exported files and
+          deletes the parent folder if it was empty.
+        Artists are free to change the file path in the ROP node.
+    """
+
+    order = pyblish.api.CollectorOrder + 0.2  # it should run after CollectFrames
+
+    hosts = ["houdini"]
+    families = [
+        "camera",
+        "ass",
+        "pointcache",
+        "imagesequence",
+        "mantraifd",
+        "redshiftproxy",
+        "review",
+        "staticMesh",
+        "usd",
+        "vdbcache"
+    ]
+    label = "Collect Files For Cleaning Up"
+
+    def process(self, instance):
+
+        import hou
+
+        node = hou.node(instance.data["instance_node"])
```

Comment:
Can we make this ignore instances without an `instance_node`? There may be 'runtime instances' from future USD implementations that are generated from other instances during publishing which themselves do not have Houdini instance node.

(Or I actually think I made the families `usdrop` (geometry) and `usdrender` (rendering) on my end)